### PR TITLE
Feature/collection

### PR DESCRIPTION
**Pull request by ekate**

### CODE REVIEW

**Comment by jgpawletko** [comment]
Hi @ekate,

You've done a lot of work here. I'm still reviewing the PR, but I have some questions:
  999. there is a database migration that creates the `collections` table, but the 
  [user story](https://www.pivotaltracker.com/n/projects/1025368/stories/87226236) indicates that the collection object should be persisted as an RDF data stream. I see that you're also using RDF data streams. I created a collection, but it doesn't look like any of the data was stored in the database.  Do you need the `collections` table?
  999. the collections form partial displays an identifier field, but this attribute is not in the [parameter whitelist](https://github.com/NYULibraries/ichabod/blob/feature/collection/app/controllers/collections_controller.rb#L82).  Do you want the user to provide this attribute? If not, you should remove the element from the view.
  999. is [this](https://github.com/NYULibraries/ichabod/pull/165/files#diff-b73bed724cb8650809575d7e058016b1R60) change needed?
  999. did you need to add in the [scaffolding stylesheet](https://github.com/NYULibraries/ichabod/pull/165/files#diff-ce495cdcf02ace6bc9ec3ce403015fcc), or could you use the existing stylesheet?

I'll probably have more comments later, but I wanted to give you some initial feedback.
Thanks,
Joe

**Comment by ekate** [comment]
@jgpawletko I've made the cjanges you've suggested. Do you have other comments ?
Thank you.
Kate

File: config/routes.rb [hunk 1]
```diff
@@ -1,12 +1,15 @@
 Ichabod::Application.routes.draw do
+  resources :collections
+
   resources :nyucores
 
   root :to => "catalog#index"
   Blacklight.add_routes(self)
 
+  get 'collections', :to => "collections#index"
```
Comment (jgpawletko) [inline config/routes.rb]: Kate, I don't think this is needed.  The `resources :collections` statement on line 2 includes this case.

File: config/initializers/.devise.rb [hunk 2]
```diff
@@ -57,6 +57,7 @@
   # :token_options = Support token authentication with options as defined in
   #                  http://api.rubyonrails.org/classes/ActionController/HttpAuthentication/Token.html
   # config.http_authenticatable = false
+  config.secret_key = ENV['SECRET_KEY_BASE']
```
Comment (jgpawletko) [inline config/initializers/.devise.rb]: Hi Kate, could you explain this change?

**Comment by jgpawletko** [comment]
Hi @ekate,
A couple of additional comments:
1.) I suggest changing the `NOID_PREFIX` constant name to something more neutral, as we're no longer using `NOIDS`.  Perhaps `ID_PREFIX`?
2.) please see comment re: explicit route for `Collections#index`.
3.) please see question re: `SECRET_KEY_BASE`
4.) please add newlines to files where newlines missing at end of file. [for example](https://github.com/NYULibraries/ichabod/pull/165/files#diff-596374a12ef429714137455430e6b4c4R39)
5.) please add some tests or delete [this](https://github.com/NYULibraries/ichabod/pull/165/files#diff-4b31828ded1b91adf26607612b09e642R1) file

Please let me know if you have any questions.
Thanks!
Best-
Joe

**Pull request closed without merging**